Scripts/Runtime/Entities/Data/ComponentDataFromSingleEntity.cs

```diff
@@ -25,7 +28,7 @@ public ComponentDataFromSingleEntity(ComponentDataFromEntity<T> lookup, Entity e
         }
 
         /// <summary>
-        /// Gets the <see cref="DynamicBuffer{T}" />. 
+        /// Gets the <see cref="DynamicBuffer{T}" />.
```

Comment:
I know this said DynamicBuffer before but that isn't correct for this struct.

Reply:
Good catch, fixed.